### PR TITLE
157002565 remove unused embeddable models

### PR DESCRIPTION
**Pull request by knowuh**
Removes unused embeddable items from portal.

### CODE REVIEW

File: db/migration-notes.txt [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  embeddable_biologica_static_organisms
+  embeddable_biologica_worlds
+)
+
+def remove_table(name)
+
+
+execute "delete from saveable_external_links where embeddable_type like '%Biologica%'"
+execute "delete from page_elements where embeddable_type like '%Biologica%'"
+execute "delete from portal_offering_embeddable_metadata where embeddable_type like '%Biologica%'"
```
Comment (scytacki) [inline db/migration-notes.txt]: Is this file needed anymore?

**Comment by knowuh** [comment]
Hello @scytacki  -- I cleaned things up as per your request.

I also rebased from master so its hard to find the fixes,  but they are all in this commit:
2f637cc
